### DISCUSSION TITLE
the movement is very slow

### DISCUSSION THREAD

**Discussion by junkfoodcafe** [sebanc/brunch#832] [Q&A]
I installed ChromeOS on a USB flash drive using Linux Mint.
But when I do USB BOOT, ChromeOS moves very slowly.
I used Windows to create a ChromeOS image and install it on a USB flash drive.
The result is the same, ChromeOS is very slow.
I use USB 3.0 for USB flash drive.
Is there a way to solve this problem?
If I installed ChromeOS on HDD, it works very well.

**Comment by Warrentheo** [comment]
USB 3.0 is the speed that the USB controller on the motherboard talks to the controller in the USB drive, but says nothing about the speed of the storage in the USB drive itself...  I would check the actual benchmark with something like Crystal Disk Mark, or the Linux "Disks" utility...  Unless you went out of your way to buy I higher speed drive, most likely that is your problem...

**Comment by GenZai** [comment]
I also tried the USB install but I'm not impressed. The system works pretty when it's not freezing, but it's not snappy, sometimes there are micro freezes, which make it not reliable. I was wondering if it was better to use a flash drive instead of a slow HDD. But it seems the HDD is more reliable.

Any one has a positive experience with the USB install ?
